src/types/index.d.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable camelcase */
+import type { GridLayerOptions, Coords, DoneCallback } from "leaflet";
+
+export type PixelValueToColorFn = (values: number[]) => string;
+
+export interface GeorasterLayerOptions extends GridLayerOptions {
+  georasters: Georaster[];
```

Comment:
Should this be `georasters?:` and `georaster?:` because user could do either one?

Reply:
I've totally re-typed this. I used the file locally in a react project and learned a few things.

Reply:
Essentially the type ensures either, `georaster` or `georasters` is defined.